src/Fantomas.CLI.Benchmarks/Runners.fs

```diff
@@ -0,0 +1,34 @@
+module Fantomas.CLI.Benchmarks.Runners
```

Comment:
We could add this into the current benchmarks project, I was just going off the initial work.

Reply:
Yeah re-using the current benchmark project make more sense to me.